modules/ducktests/tests/setup.py

```diff
@@ -53,7 +53,11 @@ def run_tests(self):
       license="apache2.0",
       packages=find_packages(),
       include_package_data=True,
-      install_requires=["ducktape==0.7.8", "requests==2.20.0"],
-      tests_require=["pytest", "mock", "monotonic"],
+      install_requires=[
+          'ducktape==0.7.8',
```

Comment:
Why we have different dependency versions here and in requirements.txt?

Reply:
Tnkx

Reply:
Fixed